test/dialects/postgres_test.py

```diff
@@ -139,3 +139,26 @@ def test_get_keywords():
     expected_result_2 = ["C", "E"]
 
     assert sorted(get_keywords(kw_list, "non-reserved")) == sorted(expected_result_2)
+
+
+@pytest.mark.parametrize(
+    "raw",
+    [
+        "select tbl1.id from tbl1 join tbl2 on tbl1.id = tbl2.id;",
+        "select tbl1.id from tbl1 join lateral tbl2 on tbl1.id = tbl2.id;",
+        (
+            "select tbl1.id from tbl1 "
+            "full outer join lateral tbl2 on tbl1.id = tbl2.id "
+            "cross join tbl3 on tbl1.id = tbl3.id "
+            "left join lateral tbl4 on tbl1.id = tbl4.id;"
+        ),
+    ],
+)
+def test_join_can_be_lateral(raw: str) -> None:
+    """Postgres can have a lateral joins."""
+    cfg = FluffConfig(
+        configs={"core": {"exclude_rules": "L009", "dialect": "postgres"}}
+    )
+    lnt = Linter(config=cfg)
+    result = lnt.lint_string(raw)
+    assert result.num_violations() == 0
```

Comment:
Again, I've wrote straightforward tests instead of using already existing here solutions. I have a lot to learn. Is there any documentation and best practices about these yml fixtures? I haven't seen that.

Reply:
Once you get used to it, it makes a lot of sense.

It's in the parser [README](https://github.com/sqlfluff/sqlfluff/blob/main/test/fixtures/parser/README.md)
which is linked from the [CONTRIBUTION.md](https://github.com/sqlfluff/sqlfluff/blob/main/CONTRIBUTING.md).

The SQL is easier to write than Python.
And the YML Parse Tree is auto generated with that command and added to Repo so we can easily see changes in the parser (e.g. if someone renames a segment type).

Reply:
Sorry for bothering you, I should've find that by myself.